carbon_tracker_workflow.ipynb

```diff
@@ -0,0 +1,2510 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ds_na = carbon_tracker.mask_region(ds_regions, ds_cb, ds_input)
</pre><p>Let's move the .sel operation which is in this function to the notebook. Or perhaps we can leave it out altogether by now (except for the time slice).</p><p>That would make this notebook able to do a worldwide model training (once we have global era5 data...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EXCITED-CO2/excited-workflow/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I left it out for now, it seems to still work ok, but let me know if that is now what you meant.